test/skate/detours/missed_stops_test.exs

```diff
@@ -37,9 +38,100 @@ defmodule Skate.Detours.MissedStopsTest do
         ]
       }
 
+      assert %MissedStops.Result{missed_stops: missed_stops} =
```

Comment:
**Nit (non-blocking)**: Could this assertion and the one below it be combined into a single assertion?